tests/unit/store/mutations.spec.ts

```diff
@@ -100,12 +103,40 @@ describe( 'mutations', () => {
 		);
 	} );
 
+	it( 'setSubclasses', () => {
+		const expectedValue = true;
+		const state: RootState = {
+			conditionRows: [ {
+				valueData: { value: 'foo', valueError: null },
+				propertyData: {
+					id: 'P123',
+					label: 'abc',
+					datatype: 'string',
+					isPropertySet: true,
+					propertyError: null,
+				},
+				propertyValueRelationData: { value: PropertyValueRelation.Matching },
+				conditionId: '0.123',
+				subclasses: true,
+			} ],
+			limit: 0,
+			useLimit: false,
+			omitLabels: true,
+			errors: [],
+		};
+
+		mutations.setSubclasses( state, { subclasses: expectedValue, conditionIndex: 0 } );
```

Comment:
This test sets the subclasses from `true` to `true`. Either the old or the new value should be `false` to actually test something.

Reply:
ah, yes, good catch.